src/app/demo/main/main.component.scss

```diff
@@ -29,3 +29,11 @@
     }
   }
 }
+
+span.draggable{
+  color:white;
```

Comment:
Please add a space after each CSS property, e.g.: `color: white;`
(for this line and below)